modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/HaskellServantCodegen.java

```diff
@@ -43,6 +43,10 @@
     protected String apiVersion = "0.0.1";
     private static final Pattern LEADING_UNDERSCORE = Pattern.compile("^_+");
 
+    public static final String PROP_SERVE_STATIC = "serveStatic";
+    public static final String PROP_SERVE_STATIC_DESC = "serve will serve files from the directory 'static'.";
+    public static final Boolean PROP_SERVE_STATIC_DEFAULT = Boolean.TRUE;
```

Comment:
Why opt-out instead of opt-in?

Reply:
I did it opt-out because I think more servers are serving static files than not. We needed that change to serve the openapi-interface. In my opinion it would be better to maybe document in the README.md how to remove the serveStatic instead of searching on how to add it if you need it.